### PR TITLE
PRE-DRAFT: rebalance zero123 hyperparams for Anya

### PR DESCRIPTION
**Pull request by claforte**
As soon as https://github.com/threestudio-project/threestudio/pull/71 is merged, I plan to rebase this, and break it down into at least 2 PRs:
1. integrate wandb
2. rebalance zero123 hyperparams, implement gradient accumulation for zero123 and reference image supervision.

### CODE REVIEW

File: threestudio/systems/zero123.py [hunk 1]
```diff
@@ -45,35 +46,45 @@ def on_fit_start(self) -> None:
...
         )
 
-    def training_step(self, batch, batch_idx):
+    def log_loss(self, name, loss_term, weight=1.0):
+        self.log(name, loss_term)
+        loss_term_weighted = weight * loss_term
+        self.log(name + "_w", loss_term_weighted)
+        return loss_term_weighted
+
+    def training_substep(self, batch, batch_idx, do_ref, loss_weight=1.0):
```
Comment (claforte) [inline threestudio/systems/zero123.py:55]: TODO: generalize to support reference-image-supervision + DeepFloyd-guidance, etc.
